bindings/python/pymongocrypt/mongocrypt.py

```diff
@@ -230,9 +240,23 @@ def __init(self):
                 self.__crypt, aes_256_ctr_encrypt, aes_256_ctr_decrypt, ffi.NULL):
             self.__raise_from_status()
 
+        if csfle_path is not None:
+             lib.mongocrypt_setopt_set_csfle_lib_path_override(self.__crypt,
+                                                               ffi.new("char[]", csfle_path.encode(
+                                                                   "UTF-8"))
+                                                               )
+        if bypass_encryption is False:
+            lib.mongocrypt_setopt_append_csfle_search_path(self.__crypt,
+                                                        ffi.new("char[]", "$SYSTEM".encode(
+                                                            "UTF-8")))
         if not lib.mongocrypt_init(self.__crypt):
             self.__raise_from_status()
 
+        if csfle_required is True and lib.mongocrypt_csfle_version_string(self.__crypt, ffi.new("uint32_t*")) \
```

Comment:
The second parameter is optional, so you can slightly simplify this here:

```suggestion
        if csfle_required is True and lib.mongocrypt_csfle_version_string(self.__crypt, ffi.NULL) \
```

Reply:
Done.